Texts/MIDs.h

```diff
@@ -1,5 +1,5 @@
 //MIDs.h
-//Generated by DRODUtil
+//Generated by DRODUTIL
```

Comment:
This change is close but not identical to the original line (case difference).  Let's revert this change back to the original line.